great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -73,30 +73,22 @@ def _get_column_cardinality(cls, df, column):
 
         return cardinality
 
-    @classmethod
-    def _get_value_set(cls, df, column):
-        partition_object = {
-            "values": ["GE_DUMMY_VAL"],
-            "weights": [1.0]
-        }
-
-        df.expect_column_kl_divergence_to_be_less_than(column, partition_object=partition_object,
-                                                   threshold=0, result_format='COMPLETE')
-
     @classmethod
     def _profile(cls, dataset):
 
 
         df = dataset
 
+        df.expect_table_row_count_to_be_between(min_value=0, max_value=None)
```

Comment:
This might be one to set as min_value=1 so there's a possibility of failing?

Reply:
+1